lib/tags/index.js

```diff
@@ -18,11 +18,14 @@ export default class TagContainer extends Component {
     const {
       tags,
       removeTag,
-      resetColour
+      resetColour,
+      resetBackgroundColour,
+      resetBorderColour
     } = this.props;
     const resetButtonStyle = {
       color: resetColour,
-      border: '1px solid' + resetColour
+      backgroundColor: resetBackgroundColour,
+      border: '1px solid' + resetBorderColour
     };
```

Comment:
Then I would probably add several clases with the specific changes, and change add one or another via props. This would let style outside structure/logic, unless there were something that changes dynamically (e.g. if we have a width = screen.width or something like this)